doc/relevancy.md

```diff
@@ -458,6 +459,26 @@ but something else still gets high traffic ([03c4bfa][], May 2015).
 If we had other signals of content quality we might be able to reduce our
 dependence on the page views.
 
+### Finding underperforming queries
+
+1. Go to the [datastudio report][]
+
+2. Apply the following filters:
+
+* `click_count >= 100`
+* `median_position = ~4`
```

Comment:
would be interested to see how you came up with the method, I think anything with a reasonably high click count and a high median is underperforming. I would recommend ignoring the mean most of the time as it can be skewed by one person clicking on result 140 for example.

Reply:
@suganya-s I copied the text from this card: https://trello.com/c/KtVv06e8/1072-document-process-to-find-underperforming-queries.

Reply:
@bilbof @laurentqro would recommend just looking at high medians and ignoring means

Reply:
Thanks @suganya-s I'll update 👌

Reply:
@suganya-s what would you say is a highish median? >4?

Reply:
yep sounds good

Reply:
if you change the `~` to a `>` this is great